projects/storefrontlib/src/layout/a11y/skip-link/directive/skip.directive.ts

```diff
@@ -0,0 +1,23 @@
+import { Directive, Input, OnDestroy, OnInit, ElementRef } from '@angular/core';
+import { SkipLinkService } from '../service/skip-link.service';
+
+@Directive({
+  selector: '[cxSkip]',
+})
+export class SkipDirective implements OnInit, OnDestroy {
+  @Input()
+  cxSkip: string;
+
+  constructor(
+    private elRef: ElementRef<any>,
```

Comment:
This seems to be an HTMLElement type always, correct? If so, we can improve the typing.